src/cephadm/cephadm

```diff
@@ -1871,6 +1871,13 @@ def command_bootstrap():
         logger.info('Setting orchestrator backend to cephadm...')
         cli(['orch', 'set', 'backend', 'cephadm'])
 
+        logger.info('Waiting for cephadm CLI to be available...')
+        def is_cephadm_available():
+            timeout=args.timeout if args.timeout else 30 # seconds
+            out = cli(['-h'], timeout=timeout)
```

Comment:
`cephadm` is always in the output of `ceph -h`, regardless of the backend (rook etc) ...

Maybe a test against the `cephadm` user ?

```suggestion
            out = cli(['cephadm', 'get-user'], timeout=timeout)
```

Reply:
in this case its the cephadm command and module that just need to be enabled... the 'ceph cephadm generate-key' command is failing because it isn't known to the mon yet.

Reply:
http://pulpito.ceph.com/sage-2020-02-18_17:45:29-rados-wip-sage-testing-2020-02-18-0802-distro-basic-smithi/#

Reply:
But, even when cephadm is not enabled, the command `ceph -h | grep cephadm` is still true ... 

```
$ cephadm bootstrap --mon-ip 127.0.0.1 --skip-ssh
...

$ cephadm shell -c ceph.conf -k ceph.client.admin.keyring
[ceph: root@host1 /]# ceph -h | grep cephadm
cephadm check-host <host> [<addr>]                                                                                                      Check whether we can access and manage a remote host
cephadm clear-key                                                                                                                       Clear cluster SSH key
cephadm clear-ssh-config                                                                                                                Clear the ssh_config file
cephadm generate-key                                                                                                                    Generate a cluster SSH key (if not present)
cephadm get-pub-key                                                                                                                     Show SSH public key for connecting to cluster hosts
cephadm get-user                                                                                                                        Show user for SSHing to cluster hosts
cephadm set-ssh-config                                                                                                                  Set the ssh_config file (use -i <ssh_config>)

[ceph: root@host1 /]# ceph cephadm get-user
Error ENOTSUP: Module 'cephadm' is not enabled (required by command 'cephadm get-user'): use `ceph mgr module enable cephadm` to enable it
```